app/javascript/modules/listingDetails/ListingDetailsPreferences.tsx

```diff
@@ -15,6 +15,7 @@ const determineDescription = (
   NRHPDistrict?: string
 ) => {
   if (customPreferenceDescription) {
+    // todo: Use GoogleCloudTranslate here
```

Comment:
question: is there a ticket we can reference in this comment, so we don't forget to implement the todo?